podman/domain/images_build.py

```diff
@@ -162,6 +165,9 @@ def _render_params(kwargs) -> Dict[str, List[Any]]:
             "squash": kwargs.get("squash"),
             "t": kwargs.get("tag"),
             "target": kwargs.get("target"),
+            'layers': kwargs.get("layers"),
```

Comment:
can you follow the double quoted format of the above items?

Reply:
Done